README.md

```diff
@@ -13,7 +13,7 @@ With localauth0 you can fake your [auth0](https://auth0.com/) tenant and test it
 In order to run localauth0 docker image execute the following:
 
 ```
-docker run -d -p 3000:3000 public.ecr.aws/prima/localauth0:0.2.0
+docker run -d -p 3000:3000 public.ecr.aws/prima/localauth0:0.2.1
```

Comment:
maybe you could start publishing the `latest` tag as well, so there is no need to mess with the README on every release... of course you could specify that there is a docker tag for every release, so people could stick to them

Reply:
Sadly this couldn't be done because actually there's not a way to overwrite tags in ecr because this option is disabled..